docs/server-request.md

```diff
@@ -17,20 +17,38 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - Read files or binary streams
 - Call APIs from same app system - use the [Call Local API](/docs/Actions/CallLocalApi.md) action instead
 
-## `Related Actions`
+## `Related actions`
 
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Parameter reference`
 
-| Parameter     | Description                           | Supports Variables |
+| Parameter     | Description                           | Supports variables |
 |---------------|---------------------------------------|--------------------|
 | URL           | Absolute URL where the resource to be invoked lives. | Yes |
```

Comment:
So it doesn't work with relative URLs?

Reply:
It works, but since we recommend the usage of Call Method Action instead, I don't think we should open up this possibility since there is a better/cleaner/faster way to do it.

Reply:
I don't understand what they have to do to one another. How would I read /Portals/0/some.json with the Call Method action?

Reply:
I see what you mean, I was referring relative URLs in relation to other ApiEndpoint methods.